src/chrome/content/rules/Tenhou.net.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	mismatch: blog.tenhou.net
+-->
+
+<ruleset name="Tenhou.net">
+	<target host="tenhou.net" />
+	<target host="cdn.tenhou.net" />
+
```

Comment:
```suggestion
	<target host="opid.tenhou.net" />

```

Reply:
I've seen this domain in Sublist3r output, but I'm not really sure what is it. It's serving empty 200 HTTP OK, this might be non-functioning private OpenID endpoint. Tenhou.net itself a online [riichi mahjong](https://en.wikipedia.org/wiki/Japanese_Mahjong) server, and I've never stumbled on this subdomain whenever I played there.

I would suggest not to add it, as it might break something and not to bloat the extension.